app/models/org.rb

```diff
@@ -151,6 +152,10 @@ class Org < ActiveRecord::Base
               count(distinct templates.family_id) as template_count,
               count(users.id) as user_count")
   }
+  
+  scope :departments, -> do
+    where(org_id: org.id)
+  end
```

Comment:
I don't think you should need to define this, it should be done auto-magically via the `has_many` relationship